apps/simulators/src/styles.scss

```diff
@@ -92,3 +92,10 @@
   display: flex;
   align-items: center;
 }
+
+biosimulations-hover-open-menu button:hover {
+  background-color: var(--primary-darker-color) !important;
+  transition: background-color 0.35s ease !important;
+  color: white !important;
+  font-weight: 700;
+}
```

Comment:
why add it here if pretty much the same thing is going in `_global.scss`?

Reply:
There are several scopes to consider when discussing repo styling: global, app-wide, local (to component), and In-line. This serves as an app-wide override for just `simulators` which will pierce the shadow-DOM only for this app...it is the property controller of navigation buttons `background-color`.